internal/flags/customflags.go

```diff
@@ -44,15 +28,7 @@ func (s *DirectoryString) Set(value string) error {
 	return nil
 }
 
-var (
-	_ cli.Flag              = (*DirectoryFlag)(nil)
-	_ cli.RequiredFlag      = (*DirectoryFlag)(nil)
-	_ cli.VisibleFlag       = (*DirectoryFlag)(nil)
-	_ cli.DocGenerationFlag = (*DirectoryFlag)(nil)
-	_ cli.CategorizableFlag = (*DirectoryFlag)(nil)
-)
```

Comment:
These 'variables' are in fact interface implementation assertions, i.e. they ensure that the type `DirectoryFlag` implements the listed interfaces. We have these as a guard because these interfaces are tested-for by the cli package at runtime.

So please do not remove these assertions.